README.md

```diff
@@ -18,12 +18,16 @@ To replicate this analysis, clone this GitHub repository, install the listed [de
 
 ```
 # download data
+python src/download_data.py --url="https://opendatatc.blob.core.windows.net/opendatatc/NCDB_2017.csv" --filepath=data/raw/NCDB_2017.csv
 
-#run eda report
+# clean and split data
+python src/clean_split_data.py --input=data/raw/NCDB_2017.csv --output=data/processed/
 
-#clean and split data
+# run eda report
+python src/eda.py --train=data/processed/NCDB_2017.csv --out_dir=results/figures/
```

Comment:
Does NCDB_2017 in processed include train set only?

Reply:
Wait, I just see below that we have train.csv as well. Should it be train.csv here as well?

Reply:
Good catch I'll change that to train.csv